plugins/amazonq/codetransform/jetbrains-community/src/software/aws/toolkits/jetbrains/services/codemodernizer/controller/CodeTransformChatController.kt

```diff
@@ -82,7 +99,7 @@ class CodeTransformChatController(
             buildCheckingValidProjectChatContent()
         )
 
-        delay(3000)
+        delay(2000)
```

Comment:
Question: What is with the delays? Also, there are some delay inconsistency in times from 3000, 2000, 500. 

If we are keeping the delays, can we at least make them a shared constant for consistence, readability and explain ability when seeing them?

Reply:
These delays were added to prevent chat item updates happening too fast that the screen looks flashy. I converted them to function to make it more consistent.